chaste_codegen/model_with_conversions.py

```diff
@@ -0,0 +1,502 @@
+import logging
+
+import cellmlmanip
+from cellmlmanip.model import DataDirectionFlow
+from cellmlmanip.rdf import create_rdf_node
+from cellmlmanip.units import UnitStore
+from pint import DimensionalityError
+from sympy import (
+    Derivative,
+    Eq,
+    Expr,
+    Float,
+    Function,
+    Pow,
+    Wild,
+    log,
+    simplify,
+    sympify,
+)
+from sympy.codegen.cfunctions import log10
+from sympy.codegen.rewriting import ReplaceOptim, log1p_opt, optimize
+
+from chaste_codegen import LOGGER
+from chaste_codegen._rdf import OXMETA, PYCMLMETA, get_variables_transitively
+
+from ._math_functions import MATH_FUNC_SYMPY_MAPPING
```

Comment:
Why two different styles of relative import for chaste_codegen?

Reply:
thanks will fix for consitency